Automattic/jetpack
# Jetpack: Add support for PPSX and similar file types

Fixes Automattic/wp-calypso#30426

#### Changes proposed in this Pull Request:

This is probably not where this function belongs, but I was able to confirm it works locally by including it in global functions. Consider this a starting point, and please let me know if and where it should be moved. :) 

This adds support for PPSX/PPTX and similar file types for Atomic sites. 

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:

* Switch to this branch
* Go to Media -> Add new file and add a PPSX, PPTX, or similar file to the media library
* The file should upload successfully!
